### ISSUE TITLE
Compare running times of NeuroML file generation with OpenWormReader vs Spreadsheet reader

### ISSUE THREAD

**Issue by mwatts15** [openworm/c302#11]
OpenWormReader will undoubtedly take longer. Must determine whether the
difference is substatial enough to merit doing query of the PyOW database
at a different phase (i.e., with a `pow translate`). This had been a
limiting factor to inclusion in c302/CElegansNeuroML previously.

blocked by #10

**Comment by pgleeson** [comment]
Currently  UpdatedSpreadsheetDataReader.py, UpdatedSpreadsheetDataReader2.py & SpreadsheetDataReader.py take under 1 sec to run, while OpenWormReader.py takes ~35sec.

**Comment by pgleeson** [comment]
No longer relevant due to move to https://github.com/openworm/ConnectomeToolbox for connectomes info
